test/test-pages/rtl-4/expected-metadata.json

```diff
@@ -2,6 +2,6 @@
   "title": "RTL Test",
   "byline": null,
   "excerpt": "Lorem ipsum dolor sit amet.",
-  "dir": "rtl",
+  "dir": null,
```

Comment:
I think we maybe could accept it for below two reasons.

1. That is expected result for the [code][code] we added to fix text direction before. The dom structure of new result is:
```
<div>
    <article dir="rtl">
        <p>...</p>
        <p>...</p>
        <p>...</p>
    </article>
</div>
```

So the final `topCandidate` for `rtl-4` test is the `<div>` node. And we check text direction from `topCandidate` and its ancestors. If we want to fix that here, we might need to check the text direction of all child nodes.

2. Even we get `null` direction, the presentation of result is still correct. Because the `<article>` node is still with `dir="rtl"` attribute.

What do you think?

[code]: https://github.com/mozilla/readability/blob/master/Readability.js#L1017-L1028